docs/source/tutorials/text.ipynb

```diff
@@ -123,6 +123,20 @@
     "raw_full_texts, full_labels = tfds.as_numpy(raw_full_ds)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "num_classes = len(set(full_labels))"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
```

Comment:
Might consider removing these as it seems specific to PyCharm IDE. might flicker back and forth if any contributor just collapses/hides this code cell.

Reply:
Good catch -- I didn't even notice Pycharm added that in. Removed.